articles/dms/howto-mysql-to-azure-mysql-powershell.md

```diff
@@ -0,0 +1,657 @@
+---
+title: "PowerShell: Run offline migration from MySQL database to Azure Database for MySQL" 
+titleSuffix: Azure Database Migration Service
+description: Learn to migrate an on-premise MySQL database to Azure Database for MySQL by using Azure Database Migration Service through Powershell script.
+services: dms
+author: "sumitgaurin"
+ms.author: "sgaur"
+manager: "arthiaga"
+ms.reviewer: "arthiaga"
+ms.service: dms
+ms.workload: data-services
+ms.custom: "seo-lt-2019"
+ms.topic: tutorial
+ms.date: 04/11/2021
+---
+
+# Migrate an on-premise MySQL database to Azure Database for MySQL by using Azure Database Migration Service through PowerShell script
+
+In this article, you migrate a MySQL database restored to an on-premises instance to Azure Database for MySQL by using the Offline migration capability of Azure Database Migration Service through Microsoft Azure PowerShell. You can migrate databases from a MySQL instance to Azure Database for MySQL instance by using the `Az.DataMigration` module in Microsoft Azure PowerShell. The article documents a collection of PowerShell scripts which can be executed in sequence to perform the offline migration of MySQL database to Azure.
+
+> [!IMPORTANT]
+> The “MySQL to Azure Database for MySQL” online migration scenario is being replaced with a parallelized, highly performant offline migration scenario from June 1, 2021. For online migrations, you can use this new offering together with [data-in replication](https://docs.microsoft.com/azure/mysql/concepts-data-in-replication). Alternatively, use open-source tools such as [MyDumper/MyLoader](https://centminmod.com/mydumper.html) with data-in replication for online migrations. 
```

Comment:
Check with Arun, but I think we know believe this is not possible until our offline migration enables migrating from a consistent snapshot on the source.  Until then they will have to use MyDumper / MyLoader along with data-in replication.

Reply:
This text is already in the online migration documentation .. so the same message is added here for consistency.